dcos_installer/backend.py

```diff
@@ -10,7 +10,6 @@
 import gen
 import gen.calc
 import gen.installer.aws
-import gen.internals
```

Comment:
I think the import was seen as not needed because the fully qualified name was being used at the use site. I can add it back in if you want, but it doesn't seem to be necessary.